sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -60,11 +57,26 @@ export default defineComponent({
     const LAST_ACTIVITY_TIME_LOCAL_STORAGE_ITEM = "lastActivityTime";
     let countdownInterval = 0;
     let idleTimeInSeconds = 0;
+    let MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT,
+      MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION,
+      MAXIMUM_IDLE_TIME_FOR_WARNING_SUPPORTING_USER,
+      MAXIMUM_IDLE_TIME_FOR_WARNING_AEST;
```

Comment:
There is no need to declare these variables since they are already declared right after.
https://github.com/bcgov/SIMS/blob/f7033322636031aa3340f960ac3b640151ad63e7/sources/packages/web/src/components/common/IdleTimeChecker.vue#L69-L74